reports/All_orders_made_last_year.sql

```diff
@@ -0,0 +1,5 @@
+SELECT o.id, c.first_name, c.last_name, o.ordered_at
+FROM orders o
+inner join customers c on c.id = o.id

```

Comment:
aqui tem uma inconsistência, cê tá fazendo um join dos customers com as orders de mesmo id, o que não tem muito valor prático. creio que deveria ser um pouco diferente:

```suggestion
inner join customers c on c.id = o.customer_id

```